barretenberg/cpp/src/barretenberg/stdlib/primitives/bigfield/bigfield.hpp

```diff
@@ -253,7 +253,7 @@ template <typename Builder, typename T> class bigfield {
      * @return bigfield
      */
     bigfield pow(const field_t<Builder>& exponent) const;
-    bigfield pow(const uint32_t exponent) const;
+    bigfield pow(const size_t exponent) const;
```

Comment:
Initially i merged with the exponent being uint32_t thinking there is a range check performed as part of the function. I think it makes more sense for this to be size_t and we let the underlying functionality report any possible overflow rather than doing size_t to uint32_t conversions in the rest of the codebase.